packages/circuit-ui/components/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -149,6 +162,7 @@ export const AutocompleteInput = forwardRef<
       allowNewItems,
       variant = 'contextual',
       'aria-setsize': ariaSetSize,
+      selectionMode = 'single',
```

Comment:
Do you foresee other selection modes being added in the future? 

If not, I would suggest making this a boolean prop called `multiple` to match the [`multiple`](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/multiple) attribute supported by the `select`, `input[type="email"]` and `input[type="file"]` HTML elements.